course_discovery/populate_course_run_reporting_type_and_announcement.py

```diff
@@ -0,0 +1,114 @@
+import argparse
+import datetime
+from dateutil.parser import parse
+import json
+import logging
+import os
+
+import logging
+
+from edx_rest_api_client.client import EdxRestApiClient
+
+
+logger = logging.getLogger()
+
+
+class CatalogApiService(object):
+    """The service to interface with edX catalog API"""
+
+    def __init__(self, oauth_host, oauth_key, oauth_secret, api_url_root):
+        try:
+            self.access_token, expires = EdxRestApiClient.get_oauth_access_token(
+                '{root}/access_token'.format(root=oauth_host),
+                oauth_key,
+                oauth_secret, token_type='jwt'
+            )
+        except Exception:
+            logger.exception('No access token acquired through client_credential flow.')
+            raise
+
+        logger.info('Token retrieved: %s', access_token)
```

Comment:
NEVER log the token. That's equivalent to logging a password in plaintext. Remove this from here and where it was copied from.